src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Log.txt

```diff
@@ -19,7 +19,7 @@
 #Error(Kind=Numeric)
 
 >> Log(-8, -2)
-#Error(Kind=Numeric)
+#Error(Kind=Numeric, Numeric)
```

Comment:
What does this mean?

Reply:
This expression generates 2 errors while being evaluated
First one is Numeric, second one too